lib/contentstack_utils/version.rb

```diff
@@ -1,3 +1,3 @@
 module ContentstackUtils
-    VERSION = "1.1.3"
+    VERSION = "1.1.4"
```

Comment:
Make it **1.2.0**. Since we are adding one more function support for the MarkType.